lib/matplotlib/backends/backend_qt.py

```diff
@@ -682,8 +682,13 @@ def _icon(self, name):
         Construct a `.QIcon` from an image file *name*, including the extension
         and relative to Matplotlib's "images" data directory.
         """
-        name = name.replace('.png', '_large.png')
-        pm = QtGui.QPixmap(str(cbook._get_data_path('images', name)))
+        # use a high-resolution icon with suffix '_large' if available
+        # note: user-provided icons may not have '_large' versions
+        path_regular = cbook._get_data_path('images', name)
+        path_large = path_regular.with_name(name.replace('.png', '_large.png'))
```

Comment:
`with_name` only accepts a new `name` as an argument
```suggestion
        path_large = path_regular.with_name(
            path_regular.name.replace('.png', '_large.png'))
```

Reply:
@daniilS Thanks!